### DISCUSSION TITLE
ESPAltherma not refreshing

### DISCUSSION THREAD

**Discussion by cjl24** [raomin/ESPAltherma#358] [General]
I managed to get ESPAltherma all hooked up and worked... for one refresh, after that it did not refresh the data in the specified refresh window of 5000 or 5s. I know it is updating as the WifiRSSI and FreeMem changes every 5 seconds correctly, but some of the heat pump data (such as outdoor air temperature) does not. If anyone could point me in the right direction, that would be great.

**Comment by johngouk** [comment]
Hi cjl24! 

What is your setup? Is it connected to Home Assistant? Which HP values are changing and which are not?

Did you change the refresh rate, it's normally 30 seconds.
